homeassistant/components/sma/sensor.py

```diff
@@ -9,9 +9,9 @@
 
 from homeassistant.components.sensor import (
     PLATFORM_SCHEMA,
-    STATE_CLASS_MEASUREMENT,
-    STATE_CLASS_TOTAL_INCREASING,
+    SensorDeviceClass,
```

Comment:
Note: this may clash with #58707 but since it's the last one on my list and #58707 hasn't progressed in a few days I decided to create the PR anyway.